src/infinitecanonicalmps.jl

```diff
@@ -249,21 +249,35 @@ function InfMPS(s::CelledVector, f::Function)
   return ψ = InfiniteCanonicalMPS(ψL, ψC, ψR)
 end
 
-function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange)
+function finite_mps(ψ::InfiniteCanonicalMPS, range::AbstractRange; ortho::String="right")
```

Comment:
It would make more sense to me to instead have a keyword argument `ortho_lims`, like the MPS constructor.

Reply:
`ortho_lims` is a bit funny because they will be off by 1 due to the environment tensors. But we can just translate 1=>2 and N=> N+1